docs/Permissions/Onchain-Permissioning/Onchain-Permissioning.md

```diff
@@ -0,0 +1,51 @@
+description: Onchain Permissioning
+<!--- END of page meta data -->
+
+# Onchain Permissioning 
+
+Onchain permissioning uses smart contracts to store and maintain the node, account, and admin account whitelists. 
+Using onchain permissioning enables all nodes to read the whitelists from one source.                                        
+
+## Permissioning Contracts
+
+The permissioning smart contracts are provided in the [PegaSysEng/permissioning-smart-contracts](https://github.com/PegaSysEng/permissioning-smart-contracts) repository: 
+
+* Ingress contracts for nodes and accounts - proxy contracts that defer the permissioning logic to the 
+Node Rules and Account Rules contracts. The Ingress contracts are deployed to static addresses. 
```

Comment:
Should we mention that they are in the genesis file?

Reply:
Done